util/missingcrypto.txt

```diff
@@ -455,6 +455,7 @@ ERR_load_PKCS12_strings
 ERR_load_PKCS7_strings
 ERR_load_RAND_strings
 ERR_load_RSA_strings
+ERR_load_OSSL_SERIALIZER_strings
```

Comment:
Why?

In fact why does this need to be public?

Reply:
It's generated by `util/mkerr.pl`, so while it's a worthy question, I'd say it's out of scope for this PR.

Reply:
We really should *not* be adding stuff to missingcrypto.txt except in exceptional circumstances. Those exceptional circumstances being we're going to come along real soon and take it out again. Either we should document this, or we should agree a way forward to make sure this isn't public by the time 3.0 goes live.

Reply:
Please create an issue for it, as this is a generic problem.

Reply:
I created a new issue (#10527)...

...and assigned you to it ;-)